packages/nx-plugin/README.md

```diff
@@ -0,0 +1,11 @@
+# nx-plugin
```

Comment:
```suggestion
# @swarmion/nx-plugin
```
This will be available on npm, could you check the other packages docs for inspiration?

It would be great if it could link to the Swarmion homepage

Reply:
keeping this open, until the documentation is complete